aup-previous.markdown

```diff
@@ -0,0 +1,154 @@
+**EXOSCALE ACCEPTABLE USE POLICY (EUSA) **
+
+Applicable starting October 12th, 2020
+
+[Previous version](https://github.com/exoscale/terms/blob/master/aup-previous.markdown)
+[Compare](https://github.com/exoscale/terms/commits/master)
+
+# 1. Introduction
+
+This Acceptable Use Policy ("AUP") describes prohibited use of the
+services provided by Akenes SA (hereafter "Exoscale") and activities
+that are not allowed on the Exoscale Services. The examples in
+this policy are not exhaustive. Exoscale may modify the AUP at any time
+and inform it's clients accordingly. If you violate the AUP, authorize
+or assist others to do so, Exoscale is entitled to suspend or terminate
+your account on the Exposcale Services as described in the
+Exoscale Terms and Conditions or Exoscale End User Service Agreement. 
+
+Capitalized terms used in this AUP and not otherwise defined herein shall have the meaning given in the Exoscale Terms and Conditions or Exoscale End User Service Agreement.
+
+# 2. No Illegal, Harmful or Offensive Use or Content
+
+You may not use, or encourage, promote, facilitate or instruct others to
+use, the Exoscale services or website for any illegal, harmful or
+offensive use, or to transmit, store, display, distribute or otherwise
+make available content that is illegal, harmful, or offensive.
+
+Prohibited activities or content include but are not limited to:
+
+-   Illegal Activities. Any illegal activities, for example, including disseminating,
```

Comment:
`for example` and `including` are the same thing.
I'd use `for example`, because you already used `including but not limited to` the sentence before.
Unless including refers to "child pornography", at which point you probably need a comma (or I'd shift it later in the line, so it's less confusing)